apps/portals-e2e/src/explore.spec.ts

```diff
@@ -274,7 +274,9 @@ const expectFacetCharts = async (page: Page) => {
         const availableFacetNames: string[] = []
         for (let i = 0; i < nAvailableFacets; i++) {
           const buttonText = await availableFacetButtons.nth(i).textContent()
-          buttonText && availableFacetNames.push(buttonText)
```

Comment:
New default lint rule requires being more explicit about these nullish checks. I think this improves readability, but I can turn it off if desired.